graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/MappedFieldTypesService.java

```diff
@@ -24,4 +24,6 @@
 
 public interface MappedFieldTypesService {
     Set<MappedFieldTypeDTO> fieldTypesByStreamIds(Collection<String> streamIds, TimeRange timeRange);
```

Comment:
It is not a part of your PR, but maybe `streamIds` should have been a Set as well.
Many usages pass Set here anyway...

Reply:
I generally prefer using `Collection<T>` instead of `List`/`Set` whenever the specific semantics are not used and I think we should make broader use of this, as it makes the method much more usable in different scenarios without having to convert the type.